CHANGELOG.md

```diff
@@ -12,9 +12,10 @@
 
 
 ### Bugfixes
--Removes inactive danger areas at EIDW [#562](https://github.com/openscope/openscope/issues/562)
+- Removes inactive danger areas at EIDW [#562](https://github.com/openscope/openscope/issues/562)
```

Comment:
Why is this line showing up as an addition?  Oh, I see, you added a space in there.